docs/data/material/getting-started/templates/checkout/Info.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import Typography from '@mui/material/Typography';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+
+const products = [
+  {
+    name: 'Professional plan',
+    desc: 'Monthly subscription',
+    price: '$15.00',
+  },
+  {
+    name: 'Dedicated support',
+    desc: 'Included in the Professional plan',
+    price: 'Free',
+  },
+  {
+    name: 'Hardware',
+    desc: 'Devices needed for development',
+    price: '$69.99',
+  },
+  {
+    name: 'Landing page template',
+    desc: 'License',
+    price: '$49.99',
+  },
+];
+
```

Comment:
```suggestion

      interface InfoProps {
        totalPrice: string;
      }
```

Let's add a prop on this component to update the value as we progress through the steps. Here's the TypeScript interface.